skia.lua

```diff
@@ -390,14 +390,7 @@ project "skia"
     "NoPCH",
   }
 
-  defines {
-    "SK_SUPPORT_GPU=0",
-    "SK_HAS_JPEG_LIBRARY",
-    "SK_HAS_PNG_LIBRARY",
-  }
-
   local t_includedirs = {
-    -- "../giflib/lib",
```

Comment:
Out of curiosity: is the removal of `giflib` related to this work, or is it a holdover from before the `skia` upgrades?

Reply:
The old skia had a giflib encoder that was enabled by Michael Adams. I don't know how the upgrade handled that but since it was commented out here and I was looking at keeping the file tidy, I removed this line.

Reply:
Ah! Gotcha. Missed that it was commented-out here, but yeah - sounds like it needed to be removed anyhow.

Reply:
Sounds like the updated Skia might have a built in gif decoder. At the least, we have not noticed any regressions with our tests that read in GIF images, so seems like it was okay to remove